src/core/hiwire.h

```diff
@@ -413,6 +422,33 @@ hiwire_get_length(JsRef idobj);
 bool
 hiwire_get_bool(JsRef idobj);
 
+bool
+hiwire_has_has_method(JsRef idobj);
+
+bool
+hiwire_has_method(JsRef idobj, JsRef idval);
+
+bool
+hiwire_has_includes_method(JsRef idobj);
+
+bool
+hiwire_includes_method(JsRef idobj, JsRef idval);
+
+bool
+hiwire_has_get_method(JsRef idobj);
+
+JsRef
+hiwire_get_method(JsRef idobj, JsRef idkey);
```

Comment:
Could you add a docstring here as well, as done below? At least one sentence saying what it does, as indeed the names can be confusing.